### PR TITLE
Update main.php

### PR DESCRIPTION
**Pull request by osworx**
Simplified setting vars, getting language vars via `$data = $this->language->all();` instead of repeating same code over again ..

### CODE REVIEW

File: install/controller/main.php [hunk 1]
```diff
@@ -1,17 +1,15 @@
 <?php
 /**
- * @package		Arastta eCommerce
+ * @package	Arastta eCommerce
  * @copyright	Copyright (C) 2015 Arastta Association. All rights reserved. (arastta.org)
- * @license		GNU General Public License version 3; see LICENSE.txt
+ * @license	GNU General Public License version 3; see LICENSE.txt
  */
 
-class ControllerMain extends Controller {
```
Comment (denisdulici) [inline install/controller/main.php:8]: I suppose this must have been removed by mistake ;)

Comment (osworx) [inline install/controller/main.php:8]: Yeah, sorry!

Comment (denisdulici) [inline install/controller/main.php:8]: Np, waiting for the new commit that fixes this.

**Comment by osworx** [comment]
updated pr

**Comment by denisdulici** [comment]
Thanks, finally, can you please fix the indents as per our standards:
https://github.com/arastta/arastta/wiki/Coding-Stardards#indentation

https://github.com/osworx/arastta/blob/patch-1/install/controller/main.php

**Comment by osworx** [comment]
This GitHub is nerving me with the intends.
Maybe I have overseen that - fixed it (the last time - no more changes if it not fits yet)

**Comment by denisdulici** [comment]
It's the same https://github.com/osworx/arastta/blob/4a21feda7035f2f22f43858959a14c8191b99001/install/controller/main.php

**Comment by osworx** [comment]
As written, have adopted the intends - it if does not work ..

**Pull request closed without merging**